der/derive/src/sequence/field.rs

```diff
@@ -188,13 +192,29 @@ impl LowerFieldEncoder {
             })
         };
     }
+
+    /// Make this field context-specific.
+    fn apply_context_specific(&mut self, tag_number: &TagNumber, tag_mode: &TagMode) {
+        let encoder = &self.encoder;
+        let number_tokens = tag_number.to_tokens();
+        let mode_tokens = tag_mode.to_tokens();
+
+        self.encoder = quote! {
+            &::der::asn1::ContextSpecific {
+                tag_number: #number_tokens,
+                tag_mode: #mode_tokens,
+                value: Clone::clone(#encoder),
+            },
```

Comment:
I think the `,` here needs to be removed:

```suggestion
            }
```

The proc macro assembles these together using `quote`:

```rust
f(&[
    #(#encode_body),*
])
```